tests/tools/oecertdump/host/sgx_quote.cpp

```diff
@@ -155,6 +159,16 @@ oe_result_t gen_report(oe_enclave_t* enclave)
                 log("Failed to verify report. result=%u (%s)\n",
                     result,
                     oe_result_str(result));
+
+                // Print TCB Info to console if verification failed
+                printf(
+                    "oe_verify_report failure (%s)\n", oe_result_str(result));
+                printf("QEID: ");
+                oe_hex_dump(quote->user_data, 16);
+                printf("CPU_SVN: ");
+                oe_hex_dump(quote->report_body.cpusvn, SGX_CPUSVN_SIZE);
+                printf("PCE_SVN: %02x\n", quote->pce_svn);
```

Comment:
This one still doesn't have a 0x prefix

Reply:
It's intentional. I'd like to see the format below as shown in the DCAP spec:
QEID: 531b7805fc5ef06ee5920cc900e2444e
CPU_SVN: 0e0e0205ff8007000000000000000000
PCE_SVN: 0a

Reply:
Can you link me to the spec?

Reply:
https://api.portal.trustedservices.intel.com/documentation#pcs-certificate